libbeat/cmd/instance/beat.go

```diff
@@ -1480,3 +1485,42 @@ func sanitizeIPs(ips []string) []string {
 	}
 	return validIPs
 }
+
+func promoteOutputQueueSettings(bc *beatConfig) error {
```

Comment:
promoteOutputQueueSettings handles the use case where the beat is started with a configuration file that has outputs defined (normal stand alone beat).